stdeb.cfg

```diff
@@ -1,5 +1,5 @@
 [DEFAULT]
-Depends3: python3-rospkg (>= 1.0.37), python3-yaml, python3-catkin-pkg (>= 0.4.0), python3-rosdistro (>= 0.4.0), python3-termcolor, python3-xmltodict, python3-rosinstall-generator, python-rosdep, python3-github, python3-docker, python3-git, git, docker-ce
+Depends3: python3-rospkg (>= 1.1.8), python3-yaml, python3-catkin-pkg (>= 0.4.0), python3-rosdistro (>= 0.7.4), python3-termcolor, python3-xmltodict, python3-rosinstall-generator, python-rosdep, python3-github, python3-docker, python3-git, git, docker-ce
```

Comment:
Please add `rospkg` to the `setup.py` and `requirements.txt` docs as well (I guess it got pulled in by a different dependency, but, as there's now a version requirement, please add it in).

Reply:
Done, thanks!